andela/AndelaEatsUI
# fix(cloudinary-bug): Fixes cloudinary spread error

Fixes cloudinary spread bug by deleting package-lock.



#### What does this PR do?

#### Description of Task to be completed?

#### How should this be manually tested?
- Pull branch
- Run `npm i`
- Run `npm run dev`
- The application should launch without any cloudinary spread error
#### What are the relevant pivotal tracker stories?
- [#story-id](pivotal-tracker-link)

#### Background Context

#### Screenshots (If Applicable)
